### PR TITLE
{devel}[GCCcore/9.3.0] wget v1.20.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#10783~~ (libidn)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 8.1.1911, x86_64, Intel(R) Xeon(R) CPU E5-2667 v3 @ 3.20GHz (haswell), Python 3.6.8
See https://gist.github.com/64a4455f234101926074a70afcd6f352 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3134.skitty.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/845a9dd3afb38653b702f5b2a8722e92 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3402.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/68bc981b1c697d75a10df281a2275211 for a full test report.

File: easybuild/easyconfigs/w/wget/wget-1.20.3-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date gnutls for security reasons
+    # ('GnuTLS', '3.6.14'),
+]
+
+# make sure pkg-config picks up system packages (OpenSSL & co)
+preconfigopts = "export PKG_CONFIG_PATH=/usr/lib64/pkgconfig:/usr/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig && "
+configopts = '--with-ssl=openssl '
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (Micket) [inline easybuild/easyconfigs/w/wget/wget-1.20.3-GCCcore-9.3.0.eb:37]: We have a variable for this nowdays!

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node105.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/0d655669ebc835d32f14e34c8b849d4a for a full test report.

File: easybuild/easyconfigs/w/wget/wget-1.20.3-GCCcore-9.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/w/wget/wget-1.20.3-GCCcore-9.3.0.eb:37]: Right, good point...

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.8.2003, x86_64, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/58a5009b855ea01158c9962bbcfde320 for a full test report.

**Pull request merged**